sigstore/_internal/fulcio/client.py

```diff
@@ -212,16 +211,19 @@ def post(
             except (AttributeError, KeyError):
                 raise FulcioClientError from http_error
 
+        if resp.json().get("signedCertificateEmbeddedSct"):
+            embeddedSct = True
```

Comment:
Tiny style nit, for consistency with the rest of the codebase (`sct_embedded` instead of `embedded_sct` to emphasize that it's a flag, not a variable holding an embedded SCT).

```suggestion
            sct_embedded = True
```

Reply:
Thanks, updated!